### ISSUE TITLE
UWP package?

### ISSUE THREAD

**Issue by knocte** [jonathanpeppers/boots#15]
I have a build failing because the file `Microsoft.Windows.UI.Xaml.CSharp.targets` is not found. I'm guessing it's because my VM (GithubAction's `windows-latest`) is missing the UWP SDK. Is there a way to use boots to install this?

**Comment by jonathanpeppers** [comment]
I think UWP is more involved than a single vsix file, and I'm not sure where to get a public download url for it?

If you need all these thing, you might be better off using Azure DevOps, as it has UWP preinstalled. DevOps is completely free for OSS projects with some limitation, I think.

**Comment by knocte** [comment]
> I think UWP is more involved than a single vsix file, and I'm not sure where to get a public download url for it?

Actually this doesn't need to be downloaded, you just need to call `vs_installer.exe` with some parameters like adviced here: https://stackoverflow.com/a/58234475/544947

> If you need all these things, you might be better off using Azure DevOps, as it has UWP preinstalled. DevOps is completely free for OSS projects with some limitation, I think.

Hehe yeah I was using AzureDevOps already myself, but I switched to GithubActions because it's more handy to have the yml directly in the repo and not need to create accounts and deal with permissions, ItJustWorksTM. Plus, I've found that GithubActions also had UWP preinstalled, just that I had an issue with my build to pick it up. More details here: https://stackoverflow.com/a/58259125/544947

Thanks!
